src/SOS/SOS.UnitTests/ConfigFiles/Unix/Debugger.Tests.Config.txt

```diff
@@ -201,7 +200,7 @@
   </Options>
 
   <DotNetDumpHost>$(DotNetRoot)/dotnet</DotNetDumpHost>
-  <DotNetDumpPath>$(RootBinDir)/bin/dotnet-dump/$(TargetConfiguration)/net6.0/publish/dotnet-dump.dll</DotNetDumpPath>
+  <DotNetDumpPath>$(RootBinDir)/bin/dotnet-dump/$(TargetConfiguration)/net8.0/publish/dotnet-dump.dll</DotNetDumpPath>
```

Comment:
Could we define the current target version as a variable so that it we can control which version we are targeting more centrally?

Reply:
I was being lazy.  I'm pretty sure I can flow the msbuild property Juan created into the config files.